app/utils/tsinterval.js

```diff
@@ -0,0 +1,13 @@
+export default function tsinterval() {
+    let d1 = new Date(this.get('gte'));
```

Comment:
time series chart is actually only the end use, the function is really just giving intervals. It probably should not have ts in its name and should rather be called "dateInterval". The consuming location can have tsInterval as the name of their values.